src/printer.rs

```diff
@@ -163,23 +163,29 @@ impl<'a> InteractivePrinter<'a> {
             panel_width = 0;
         }
 
-        let highlighter = if input
+        let highlighter_and_syntax_set = if input
```

Comment:
is it possible to deconstruct the tuple here into separate variables? Does it make sense to do it? My reasoning is just to make lines 200/201 prettier instead of referring to `.0` and `.1`.

Reply:
Good idea! Will fix

Reply:
This is fixed now and is a clear improvement.